os/board/common/partitions.c

```diff
@@ -227,4 +237,38 @@ int configure_mtd_partitions(struct mtd_dev_s *mtd, struct partition_data_s *par
 
 	return OK;
 }
+
+void automount_fs_partition(partition_info_t *partinfo)
```

Comment:
The main reason for these changes is generalization of userfs and romfs mounting code?
Or is it also intended to serve some other specific purpose?

Reply:
It is for generalization of userfs and romfs.
Do you have any idea?

Reply:
We have to scan the partition list in config_mtd_partitions to get "partno" and "minor".
We also need to do that when we use dump_tool to extract primary or external flash partition.

Reply:
If you want to expand this function, you can do after merging this.
If possible to use common, it is good for most of the cases.